pkg/provider/cloud/openstack/provider.go

```diff
@@ -246,10 +246,16 @@ func (os *Provider) InitializeCloudProvider(cluster *kubermaticv1.Cluster, updat
 			Build().
 			NodePorts()
 
+		allowedIPRange := cluster.Spec.Cloud.Openstack.AllowedIPRange
+		if allowedIPRange == "" {
+			allowedIPRange = "0:0:0:0/0"
```

Comment:
Same here, shouldn't this be `0.0.0.0/0`? Or is this some ipv6 notation I'm not aware of?